scripts/release.ts

```diff
@@ -182,27 +197,20 @@ function diffJson(from: string = "latest", to?: string): string {
       "web-features",
       "data.json",
     );
-    const prettyJson = execSync(`jq . "${pkgJson}"`, {
-      encoding: "utf-8",
-    });
-    const fp = join(temporaryDir, `data.${version}.json`);
-    writeFileSync(fp, prettyJson);
-    return fp;
+
+    return toPrettyJson(pkgJson, join(temporaryDir, `data.${version}.json`));
   }
 
-  const fromFp = pkgToJsonFile(from);
-  const toFp: string = (() => {
+  const fromFp = await pkgToJsonFile(from);
+  const toFp: string = await (() => {
     if (to) {
       return pkgToJsonFile(to);
     } else {
       build();
-      const preparedJson = join(packages["web-features"], "data.json");
-      const prettyPreparedJson = execSync(`jq . "${preparedJson}"`, {
-        encoding: "utf-8",
-      });
-      const fp = join(temporaryDir, "data.HEAD.json");
-      writeFileSync(fp, prettyPreparedJson);
-      return fp;
+      return toPrettyJson(
+        join(packages["web-features"], "data.json"),
+        join(temporaryDir, "data.HEAD.json"),
```

Comment:
(stylistic comments, feel free to ignore)

I liked the less-compact version b/c was more explicit, and easier to read.

My preference is to put all file paths/names/parts in consts at top of file, or at least function.

Reply:
(I ended up reverting this and starting over. Marking it as resolved.)